README.md

```diff
@@ -20,7 +20,7 @@
 - [Route Parameter Hooks](#route-parameter-hooks)
 - [PHP Templates](#php-templates)
 - [Configurations](#configurations)
-- [Headers, Cookies, Session, Uploads](#headers-cookies-session-uploads)
+- [Headers, Cookies, Session, Uploads](#user-content-headers-cookies-session-and-uploads)
```

Comment:
Hi Kanti. It should be `#headers-cookies-session-and-uploads`. Please update it so I can merge it in. Thanks.